timemachine/ff/handlers/nonbonded.py

```diff
@@ -226,12 +231,9 @@ def apply_bond_charge_corrections(initial_charges, bond_idxs, deltas):
     assert len(deltas) == len(bond_idxs)
 
     net_charge = jnp.sum(initial_charges)
-    net_charge_is_integral = jnp.isclose(net_charge, jnp.round(net_charge), atol=1e-5)
```

Comment:
These are not integral when the compound is charged as we multiply by sqrt(ONE_4PI_ESP0). I moved the integral check to the charging stage as that seemed conceptually cleaner.